packages/orbit-components/src/Seat/index.js

```diff
@@ -1,45 +1,83 @@
 // @flow
-import styled from "styled-components";
+import styled, { css } from "styled-components";
 import * as React from "react";
 
 import Stack from "../Stack";
 import Text from "../Text";
 import randomID from "../utils/randomID";
 import defaultTheme from "../defaultTheme";
-import SeatItem from "./SeatItem";
+import SeatNormal, {
+  StyledPath as StyledPathNormal,
+  StyledStrokeNormal,
+} from "./components/SeatNormal";
+import SeatSmall, {
+  StyledPath as StyledPathSmall,
+  StyledStrokeSmall,
+} from "./components/SeatSmall";
+import {
+  resolveHoverColor,
+  resolveFillColor,
+  resolveFocusColor,
+  resolveCloseIconColor,
+} from "./components/helpers";
+import SeatCircle, { StyledCirclePath } from "./components/SeatCircle";
 import { SIZE_OPTIONS, TYPES } from "./consts";
 
 import type { Props } from "./index";
 
-const getSize = ({ size, selected }) => {
+const getSize = ({ size }) => {
   const height = {
-    [SIZE_OPTIONS.EXTRASMALL]: "11px",
-    [SIZE_OPTIONS.SMALL]: selected ? "43px" : "36px",
-    [SIZE_OPTIONS.MEDIUM]: selected ? "54px" : "46px",
+    [SIZE_OPTIONS.SMALL]: "36px",
+    [SIZE_OPTIONS.MEDIUM]: "46px",
   };
 
   const width = {
-    [SIZE_OPTIONS.EXTRASMALL]: "10px",
-    [SIZE_OPTIONS.SMALL]: selected ? "40px" : "34px",
-    [SIZE_OPTIONS.MEDIUM]: selected ? "54px" : "47px",
+    [SIZE_OPTIONS.SMALL]: "32px",
+    [SIZE_OPTIONS.MEDIUM]: "46px",
   };
 
   return `width: ${width[size]}; height: ${height[size]};`;
 };
 
-const resolveViewBoxSize = (size, selected) => {
-  if (size === SIZE_OPTIONS.EXTRASMALL) return "0 0 10 11";
-  if (selected) return size === SIZE_OPTIONS.SMALL ? "0 0 40 43" : "0 0 54 54";
-  return size === SIZE_OPTIONS.SMALL ? "0 0 34 36" : "0 0 47 46";
+const StyledSeatWrapper = styled.div`
+  ${({ type, selected, theme }) => css`
+    position: relative;
+    cursor: pointer;
+    outline: none;
+    &:hover {
+      ${StyledPathNormal}, ${StyledPathSmall}, ${StyledStrokeSmall} {
+        transition: fill ${theme.orbit.durationFast} ease-in;
```

Comment:
One tiny thing: to me it looks better if the transition effect is present both when the mouse enters and when it leaves. This way when the mouse leaves the color switches back instantaneously, which feels a bit choppy, but that's just my taste.